actors/verifreg/tests/verifreg_actor_test.rs

```diff
@@ -0,0 +1,240 @@
+#![deny(unused_must_use)] // Force unwrapping Result<_, Err>
+
+use fvm_shared::{HAMT_BIT_WIDTH, MethodNum};
+use fvm_shared::address::Address;
+use fvm_shared::bigint::bigint_ser::BigIntDe;
+use fvm_shared::encoding::RawBytes;
+use lazy_static::lazy_static;
+
+use fil_actor_verifreg::{Actor as VerifregActor, AddVerifierClientParams, AddVerifierParams, DataCap, Method, MINIMUM_VERIFIED_DEAL_SIZE, State};
+use fil_actors_runtime::{ActorError, make_empty_map, make_map_with_root_and_bitwidth, SYSTEM_ACTOR_ADDR};
+use fil_actors_runtime::test_utils::*;
+
+lazy_static! {
+    static ref ROOT_ADDR: Address = Address::new_id(101);
+    static ref VERIFIER_ALLOWANCE: DataCap = MINIMUM_VERIFIED_DEAL_SIZE.clone() + DataCap::from(42);
+}
+
+fn construct_runtime() -> MockRuntime {
+    MockRuntime {
+        receiver: *ROOT_ADDR,
+        caller: *SYSTEM_ACTOR_ADDR,
+        caller_type: *SYSTEM_ACTOR_CODE_ID,
+        ..Default::default()
+    }
+}
+
+fn make_harness() -> (Harness, MockRuntime) {
+    let mut rt = construct_runtime();
+    let h = Harness { root: *ROOT_ADDR };
+    h.construct_and_verify(&mut rt, &h.root);
+    return (h, rt);
+}
+
+mod construction {
```

Comment:
we can also go the each (test) module its own file route.
I also considered nested modules but decided against due to the noise.

Reply:
Yes, I noted in my description that multiple files is probably better for larger actors. I think it's fine to leave to dev discretion.